src/frontend/sharedComponents/PasscodeInput.tsx

```diff
@@ -77,6 +77,7 @@ export const PasscodeInput = React.forwardRef<TextInput, PasscodeInputProps>(
     return (
       <CodeField
         {...codeFieldProps}
+        testID="SETTINGS.passcode-inp"
```

Comment:
think this component should specify a prop and pass that here since this is a shared component and theoretically could be used on any screen, not just settings-related screens

Reply:
Ok passed a prop.